src/components/basic/SelectList/index.tsx

```diff
@@ -81,10 +94,8 @@ export const SelectList = ({
       options={items.map((item: any) => item)}
       // eslint-disable-next-line
       getOptionLabel={(option) => option[keyTitle] || ''}
-      onChange={(_, reason) => {
-        setSelected(reason)
-
-        onChangeItem(reason)
+      onChange={(_event, _nextValue) => {
+        handleChange(_nextValue)
```

Comment:
please only use names starting with underscore if the variable is not used
`_nextValue` change to `nextValue`

Reply:
Code updated